Resources/Prototypes/Entities/Structures/Storage/Canisters/gas_canisters.yml

```diff
@@ -360,7 +360,7 @@
   components:
     - type: Sprite
       layers:
-        - state: redws
+        - state: freon
```

Comment:
this should just have the actual miasma canister ported instead of using freon's (since I am going to be implementing freon as well)